src/api/nodeRpcCall.js

```diff
@@ -964,6 +964,34 @@ const getLegislation = async (tier) => {
   return legislationById;
 };
 
+const getOfficialRegistryEntries = async () => {
```

Comment:
I think we should split this.

GetCompanyRegistryEntries should be most of the logic
GetOfficialCompanyRegistryEntries should call the GetCompanyRegistryEntries function and pass it a 0 - the index of official registry